docs/src/explanations/chisel-type-vs-scala-type.md

```diff
@@ -47,28 +47,36 @@ class MyModule(gen: () => MyBundle) extends Module {
     val xReg:     MyBundle     = Reg(new MyBundle(3))       //      x          -
     val xIO:      MyBundle     = IO(Input(new MyBundle(3))) //      x          -
     val xRegInit: MyBundle     = RegInit(xIO)               //      x          -
-    val xLit:     MyBundle     = xType.Lit(                 //      x          x 
-      _.foo -> 0.U(3.W), 
+    val xLit:     MyBundle     = xType.Lit(                 //      x          x
+      _.foo -> 0.U(3.W),
       _.bar -> 0.U(3.W)
     )
     val y:        MyBundle = gen()                          //      ?          ?
-    
+
     // Need to initialize all hardware values
     xReg := DontCare
 }
 ```
 
 ```scala mdoc:invisible
 // Just here to compile check the above
-ChiselStage.elaborate(new MyModule(() => new MyBundle(3)))
+def elaborate(module: => chisel3.RawModule) = {
```

Comment:
again this one could have just stayed as chisel3.stage.ChiselStage.elaborate(...)?

In the end I assume our plan is that chisel3.stage.ChiselStage will NOT have an emitSystemVerilog, and/or circt.stage will have all these same functions? If the latter is the plan, this PR could have just added a .elaborate() to circt.stage?